Misc/NEWS.d/next/Library/2017-09-18-10-57-04.bpo-31499.BydYhf.rst

```diff
@@ -0,0 +1,3 @@
+xml.etree: xmlparser_gc_clear() now sets self.parser to NULL to prevent a
```

Comment:
Too much implementation details. They are good on the tracker or in commit message, but not in a changelog. Look at this from the point of view of common Python user.

Reply:
Done, I simplified the NEWS entry.